### DISCUSSION TITLE
Low profile keys not registering

### DISCUSSION THREAD

**Discussion by eeJoe** [Yowkees/keyball#321] [Q&A]
Hi Yowkees,

I have made the purchase of Keyball 61 last month and nearly close to finish building it (I seem to have an issue with one of the pro micro board on right side of the keyboard, so hopefully that's resolved after I receive the new board).
 
At the moment, one of the issues I'm having are in relation to the low profile switches I've ordered (Kailh Low Profile Choc Switches Brown - https://chosfox.com/collections/mechanical-keyboard-switch/products/kailh-low-profile-choc-switches?variant=42514647646402). 
![Photo 28-8-2023, 10 36 19](https://github.com/Yowkees/keyball/assets/64901119/7932426e-339f-4f8d-87a4-eeadff06bfa0)

Based on the photo, is seems like the stems from the switches are falling a little short, not quite getting the full contact they need with the sockets installed on the PCB. I did ensure to test with tweezers the keys register correctly prior to installing the switches so definitely not a soldering issue.

Would there be any easy way to workaround this issue if you can suggest any? If not, can you recommend a few options to get the correct set of switches that will work with Keyball 61 please?

Appreciate if you can suggest vendors shipping to Australia. Thanks in advance!

Joe

**Comment by Yowkees** [comment]
Hi Joe,

If you want to use Low profile switches for thumb keys, you have to remove middle acryl plates.
And change metal spacer from 7mm to 4mm (Short spacers are included in Keyball61 kit).
See the images below for examples of correct installation. Thank you. Yowkees

![Low_thumb](https://github.com/Yowkees/keyball/assets/84209772/acf9cd0d-6174-40c5-9386-c23ed2614b49)

**Reply by eeJoe** [reply]
Appreciate your timely reply Yowkees!
I have followed your instruction & checked that it works (the side without Promicro issue).

Another question if you don't mind... I note in your build guide that the promicro boards are **NOT** to be soldered, but when I search through Q&A with other ppl's questions & photos, they all seem to have soldered the promicro boards, should I solder them as well? 
![Photo 31-8-2023, 11 51 17](https://github.com/Yowkees/keyball/assets/64901119/bef9e373-36d2-4991-8552-64f062b2e0b5)

Best,

Joe
